other/allowed_pod_priorities.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: allowed-podpriorities
+  annotations:
```

Comment:
Since we have all three rules defined, should we disable auto-gen by the annotation? Once https://github.com/kyverno/kyverno/pull/1914 is merged, we can remove the other two rules for pod controllers.

Reply:
But I thought the auto-gen controller wouldn't apply in those situations because both policies involve `validate.deny` conditions.

Reply:
When I say "wouldn't apply" I mean even after that PR gets merged. The PR doesn't provide a provision for `validate.deny` conditions, or am I wrong?

Reply:
You are right! I was focus on the path conversion in auto-gen. It looks good!

Reply:
This is why we really need [this functionality](https://github.com/kyverno/kyverno/issues/1933). I think that's the last remaining gap for auto-gen.